events/guild_message_poll_events.go

```diff
@@ -0,0 +1,31 @@
+package events
+
+import (
+	"github.com/disgoorg/disgo/discord"
+	"github.com/disgoorg/snowflake/v2"
+)
+
+// GenericGuildMessagePollVote is called upon receiving GuildMessagePollVoteAdd or GuildMessagePollVoteRemove (requires gateway.IntentGuildMessagePolls)
+type GenericGuildMessagePollVote struct {
```

Comment:
do we want some more cache getters?
`Channel() (discord.GuildMessageChannel, bool)`
`Member() (discord.Member, bool)`

Reply:
channel sure, member probably not